.github/workflows/test.yml

```
@@ -56,3 +56,68 @@ jobs:
         echo ---
         read -ra ORAS_VERSION_INSTALLED <<<$(oras version)
         [ "${ORAS_VERSION_INSTALLED[1]}" == "$ORAS_VERSION_EXPECTED" ]
+
+  create-test-variables:
+    runs-on: ubuntu-latest
+    outputs:
+      url: ${{ steps.get-url.outputs.URL }}
+      checksum: ${{ steps.get-checksum.outputs.CHECKSUM }}
+    steps:
+      - id: checkout
+        uses: actions/checkout@v3
+      - id: get-url
+        run: echo "URL=$(jq -r 'keys_unsorted[0] as $k | .[$k].linux.amd64.url' src/lib/data/releases.json)" >> "$GITHUB_OUTPUT"
+      - id: get-checksum
+        run: echo "CHECKSUM=$(jq -r 'keys_unsorted[0] as $k | .[$k].linux.amd64.checksum' src/lib/data/releases.json)" >> "$GITHUB_OUTPUT"
```

Suggest to read the file only once and merge the steps
```suggestion
      - id: get-checksum-url
        run: |
        RELEASE=$(jq -r 'keys_unsorted[0] as $k | .[$k].linux.amd64' src/lib/data/releases.json)
        echo "CHECKSUM=$(echo $RELEASE | jq -r '.checksum')" >> "$GITHUB_OUTPUT"
        echo "URL=$(echo $RELEASE | jq -r '.url')" >> "$GITHUB_OUTPUT"
```